README instructions result in compile error

I'm trying to compile an empty rust project (just a Cargo.toml and a main.rs that contains nothing more than the code in the README) and I'm getting a compile error.

llamaforge:mmdb michaeldehaan$ cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading serde v1.0.17
   Compiling serde v1.0.17
   Compiling mmdb v0.1.0 (file:///Users/michaeldehaan/Devel/mmdb)
error[E0463]: can't find crate for `serde_derive`
 --> src/main.rs:2:1
  |
2 | extern crate serde_derive;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error(s)

--- a/README.md
+++ b/README.md
@@ -20,9 +20,22 @@ You may be looking for:
 
 ## Serde in action
 
+**Cargo.toml**:
+
+```toml
+[dependencies]
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+```
+
+Click on the "Run" button to have a quick taste:
+
 <a href="http://play.integer32.com/?gist=9003c5b88c1f4989941925d7190c6eec" target="_blank">
 <img align="right" width="50" src="https://raw.githubusercontent.com/serde-rs/serde-rs.github.io/master/img/run.png">
 </a>
+
+**src/lib.rs**:
 
 ```rust
 #[macro_use]

